manim/animation/animation.py

```diff
@@ -138,7 +142,8 @@ def copy(self) -> "Animation":
 
     # Methods for interpolation, the mean of an Animation
     def interpolate(self, alpha: float) -> None:
-        alpha = np.clip(alpha, 0, 1)
+        # alpha = np.clip(alpha, 0, 1)
+        alpha = min(max(alpha, 0), 1)
```

Comment:
Is there any difference between the two methods?

Reply:
one returns an np.ndarray (with a single number) the other returns a float.

Reply:
I think NumPy in terms of performance is faster right? So, will it cause any problems with speed?

Reply:
we are using it on a single number - an array of length 1.